test_programs/execution_success/double_verify_proof/src/main.nr

```diff
@@ -2,31 +2,34 @@ use dep::std;
 
 fn main(
     verification_key: [Field; 114],
-    proof: [Field; 94],
+    // This is the proof without public inputs attached.
+    // 
+    // This means: the size of this does not change with the number of public inputs.
+    proof: [Field; 93],
     public_inputs: [Field; 1],
+    // This is currently not public. It is fine given that the vk is a part of the circuit definition.
+    // I believe we want to eventually make it public too though.
     key_hash: Field,
-    input_aggregation_object: [Field; 16],
-    proof_b: [Field; 94]
-) -> pub [Field; 16] {
-    let output_aggregation_object_a = std::verify_proof(
+    proof_b: [Field; 93]
+) {
+    // Create dummy aggregation object
+    // It is no longer needed but we don't want to make a breaking change to the serialization format
+    // just yet. The object can be anything as the backend is no longer checking it anyways.
+    let unused_input_aggregation_object = [verification_key[0]];
```

Comment:
This is probably the cause. Not sure what changes have been made in the monorepo but this will likely break against the `bb` version we're working with.

Reply:
![image](https://github.com/noir-lang/noir/assets/5372114/21ad1868-d487-4623-8d3a-d5d8856d79e8)

Reply:
Not really, we just need to have a compatible published release of `bb` for nargo to target when changes are pushed back into the main Noir repository (this could even be just a nightly version). This ensures that you can always compile nargo on any mainline commit in this repository and automatically download a compatible `bb` version.

If we were to monorepo Noir to live with barretenberg then either
- we maintain this slight delay on updating the version of `bb` we target and update when necessary/convenient. (i.e. status quo)
- we automatically publish a version of `bb` for every commit to master and have nargo pull the version from that commit.
- break the guarantee that `nargo` will always download a compatible version of `bb`.

Reply:
Should we just undo the changes to the double verify proof here, and readd them back on the aztec-packages repo as a temporary delay of getting this in? I think BB won't be released until tuesday

Reply:
Yeah, I think that would be the simplest way to handle this for the time being. I'm not sure on the frequency the subrepo is updated (and so how often merge conflicts would need to be addressed) but it allowing a divergence in tests until tuesday doesn't seem to be that bad.